media-sound/whipper/metadata.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>gentoo@retornaz.com</email>
+		<name>Quentin Retornaz</name>
+	</maintainer>
+	<maintainer type="person">
+		<email>gentoo@wizzup.org</email>
+		<name>Merlijn Wajer</name>
+		<description>Whipper project developer</description>
```

Comment:
I presume you've gotten his permission for that.

Reply:
Yes, he gave me his permission on the project IRC channel on freenode #whipper, if you wish to ask him yourself.

Reply:
Yep, all good.